src/ResponseTracingFilter.cs

```diff
@@ -0,0 +1,57 @@
+using System.Diagnostics;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.Options;
+
+namespace Byndyusoft.AspNetCore.Instrumentation.Tracing
+{
+    public class ResponseTracingFilter : IAsyncResultFilter
+    {
+        private static readonly object None = new();
+        private readonly AspNetMvcResponseTracingOptions _options;
+
+        public ResponseTracingFilter(IOptions<AspNetMvcResponseTracingOptions> options)
+        {
+            _options = options.Value;
+        }
+
+        public async Task OnResultExecutionAsync(
+            ResultExecutingContext context,
+            ResultExecutionDelegate next)
+        {
+            await next();
+
+            if (Activity.Current == null)
+                return;
+
+            var tags = new ActivityTagsCollection
+            {
+                {"http.response.header.content_type", context.HttpContext.Response.ContentType},
+                {"http.response.header.content_length", context.HttpContext.Response.ContentLength}
+            };
+
+            if (TryGetBody(context.Result, out var body))
+                tags.Add("http.response.body", JsonSerializer.Serialize(body, _options.JsonSerializerOptions));
```

Comment:
This can cause problems, I guess because tags are usually small, but response can be large. Why you don't use logs?

Reply:
Activity.Events are converted into Span.Logs inside jaeger.

Reply:
okay, it looks nice)